docs/index.md

```diff
@@ -2525,6 +2530,42 @@ object will read the local file instead.
       'timeseries/51922d0b37203f2a8c000010', connection);
 ```
 
+External Connectors
+-------------------
+
+BigML offers an `externalconnector` resource that can be used to connect to
+external data sources. The description of the API requirements to create an
+`ExternalConnector` can be found at the
+[API documentation](https://bigml.com/api/externalconnectors). The compulsory
+information to create an external connector are parameters like the type
+of database manager, the host, user, password and table that we need to access.
+This must be provided as first argument to the `bigml.ExternalConnector`
+create method or can be set in environment variables:
+
+```batch
+export BIGML_EXTERNAL_CONN_HOST=db.host.com
+export BIGML_EXTERNAL_CONN_PORT=4324
+export BIGML_EXTERNAL_CONN_USER=my_user
+export BIGML_EXTERNAL_CONN_PWD=my_password
+export BIGML_EXTERNAL_CONN_DB=my_database
+export BIGML_EXTERNAL_CONN_SOURCE="postgresql"
+```
+
+
+```js
+    var bigml = require('bigml');
+    var externalConnectorConn = new bigml.ExternalConnector(),
+      externalConnectorId, connectionInfo,
+      args = {"name": "my connector"}, retry = false;
+    externalConnectorConn.create(connectionInfo, args, retry,
```

Comment:
I think it would be useful here to show how you inutilizare `connectionInfo`

Reply:
inutilizare?

Reply:
The connectionInfo in this case is set to undefined but internally the class retrieves the info in the environment variables.

Reply:
that should have been initialise -- don't know how it morphed into "inutilizare"...